### PR TITLE
Replaced Broken Link in docs/source/about/third_party_packages.rst

### PR DESCRIPTION
**Pull request by justindhillon**
## Description
In the file docs/source/about/third_party_packages.rst, there is a link to https://packages.aosc.io/packages/sunshine. As of the date this pull request is made, this is a dead link. So I replaced it with https://aosc-packages.cth451.me/packages/sunshine. I used [link-inspector](https://github.com/justindhillon/link-inspector) to find this broken link.

## Type of Change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Dependency update (updates to dependencies)
- [x] Documentation update (changes to documentation)
- [ ] Repository update (changes to repository files, e.g. `.github/...`)

## Checklist
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have added or updated the in code docstring/documentation-blocks for new or existing methods/components

## Branch Updates
LizardByte requires that branches be up-to-date before merging. This means that after any PR is merged, this branch
must be updated before it can be merged. You must also
[Allow edits from maintainers](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork).
- [x] I want maintainers to keep my branch updated

### CODE REVIEW

File: docs/source/about/third_party_packages.rst [hunk 1]
```diff
@@ -8,7 +8,7 @@ AOSC
...
    :alt: AOSC Version
-   :target: https://packages.aosc.io/packages/sunshine
+   :target: https://aosc-packages.cth451.me/packages/sunshine
```
Comment (ReenigneArcher) [inline docs/source/about/third_party_packages.rst:11]: Probably just remove this entire item. It's no longer available here: https://repology.org/project/sunshine/versions

Which is where we were pulling the version from.

**Review by ReenigneArcher — CHANGES_REQUESTED** [review]

File: docs/source/about/third_party_packages.rst (hunk 1 above)
Comment (justindhillon) [reply docs/source/about/third_party_packages.rst]: I made another commit that removes AOSC. It is now completely removed from docs/source/about/third_party_packages.rst

Comment (ReenigneArcher) [reply docs/source/about/third_party_packages.rst]: Thanks!

**Comment by ReenigneArcher** [comment]
Please put these changes into #1813

**Pull request closed without merging**